src/portal/lib/src/service/interface.ts

```diff
@@ -290,25 +290,35 @@ export enum VulnerabilitySeverity {
 
 export interface VulnerabilityBase {
   id: string;
-  severity: VulnerabilitySeverity;
+  severity: string;
   package: string;
   version: string;
 }
 
 export interface VulnerabilityItem extends VulnerabilityBase {
-  link: string;
-  fixedVersion: string;
+  links: string[];
+  fix_version: string;
   layer?: string;
   description: string;
 }
 
 export interface VulnerabilitySummary {
-  image_digest?: string;
-  scan_status: string;
-  job_id?: number;
-  severity: VulnerabilitySeverity;
-  components: VulnerabilityComponents;
-  update_time: Date; // Use as complete timestamp
+    report_id?: string;
```

Comment:
Are you sure this report_id type is not number? If yes, that will be fine.

Reply:
it‘s string type